### ISSUE TITLE
Add Amazon

### ISSUE THREAD

**Issue by MeNsaaH** [bisoncorps/search-engine-parser#44]

**Comment by kaustavbhattacharya07** [comment]
Hello! I am interested in adding this enhancement. Is the requirement something related to extracting the top ten results for a particular search? Can you kindly tell me regarding the requirement?

**Comment by MeNsaaH** [comment]
Yeah. Given a search on Amazon, it should return the titles, description, links, price and ratings for the search. Check out the [Contribution guide](https://github.com/bisoncorps/search-engine-parser/blob/master/CONTRIBUTING.md) for more details on contribution

**Comment by devajithvs** [comment]
I tried adding amazon. They have a stringent policy against web scraping. Every request returns an HTML page below:

To discuss automated access to Amazon data, please contact api-services-support@amazon.com.
For information about migrating to our APIs, refer to our Marketplace APIs at https://developer.amazonservices.com/ref=rm_c_sv, or our Product Advertising API at https://affiliate-program.amazon.com/gp/advertising/api/detail/main.html/ref=rm_c_ac for advertising use cases.

Enter the characters you see below
Sorry, we just need to make sure you're not a robot. For best results, please make sure your browser is accepting cookies

**Comment by MeNsaaH** [comment]
Wow, that's some serious stuff. Maybe we'll look into adding a selenium driver. That should work, I think @devajithvs

**Comment by devajithvs** [comment]
Tried that too.

**Comment by MeNsaaH** [comment]
Alright, I think we'll have to look into options of using selenium libraries tho. Meanwhile, I'll try out some additional headers and see where it gets us
